sdk-extensions/autoconfigure-spi/src/main/java/io/opentelemetry/sdk/autoconfigure/spi/internal/DefaultConfigProperties.java

```diff
@@ -210,6 +210,8 @@ public static Set<String> getSet(ConfigProperties config, String name) {
   public Map<String, String> getMap(String name) {
     return getList(ConfigUtil.normalizePropertyKey(name)).stream()
         .map(keyValuePair -> filterBlanksAndNulls(keyValuePair.split("=", 2)))
+        // Filter entries with an empty value, i.e. "foo="
+        .filter(splitKeyValuePairs -> !(splitKeyValuePairs.size() == 1))
```

Comment:
```suggestion
        .filter(splitKeyValuePairs -> splitKeyValuePairs.size() != 1)
```
or
```suggestion
        .filter(splitKeyValuePairs -> splitKeyValuePairs.size() > 1)
```

Reply:
Turns we limit this to a size of 2 when we call `split("=", 2)`

Adjusted the logic to fail if entries don't have at least one `=`, or if the the key is empty, but succeed if the value is empty.

Reply:
LGTM 👍